routes/web.php

```diff
@@ -18,3 +18,6 @@
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
+Route::get('/default', function () {
+    return view('pages.default');
+});
```

Comment:
It's always a good practice to leave a blank line at the end of every file.

Reply:
@reifred when do  we draw the line at nitpicking?

Reply:
I am sorry for this comment. Don't add the line.

Reply:
@reifred I have added the line.